docker/cyclecloud8/scripts/cyclecloud_install.py

```diff
@@ -418,7 +418,7 @@ def modify_cs_config(options):
                     new_config.write(line)
 
             if 'webServerHostname' in options and options['webServerHostname']:
-                new_config.write('webServerHostname={}\n'.format(options['webServerHostname']))
+                new_config.write('\n\nwebServerHostname={}\n'.format(options['webServerHostname']))
```

Comment:
Instead,
I suggest adding the newlines before the IF statement independent of whether we're setting webServerHostname (otherwise this problem will happen again with the next property we treat this way)

Reply:
ok yeah makes sense